shared/desktop/yarn-helper/log-to-trace.tsx

```diff
@@ -196,18 +204,20 @@ lines.forEach(line => {
       lastGuiLine = info
       break
     default:
-      console.log('Unknown line type:', info.type, ':', line)
+    // Kinda noisy, off for now
```

Comment:
This might be a little less noisey with the above regex change.

Reply:
its actually noisy cause there's a bunch of lines that aren't delimited correctly (+-| etc) so you get a lot of other chars